.github/pull_request_template.md

```diff
@@ -0,0 +1,24 @@
+# Description
+Please include a summary of the change and which issue is fixed or what feature is added. Please also include relevant motivation and context. List any dependencies that are required for this change.
+
+## Type of change
```

Comment:
Just noticed this header is inconsistent with the rest, not sure if that was intentional or not but just a cosmetic comment